### ISSUE TITLE
tensorflow Load model gives the random prediction in python new session.(LSTM.)

### ISSUE THREAD

**Issue by samrajkodai** [keras-team/tf-keras#421]
after train and test the model gives the good accuracy and perfect predictions.

model.save()
load_model()
once i save the model and testing in a new python session it gives random prediction.

i tried many methods but I'm not able to solve the issue. anyone have a solution?

i have attached the code also
my tensorflow version is  2.11.0-dev20220812
[issue.txt](https://github.com/tensorflow/tensorflow/files/9597363/issue.txt)
